src/Umbraco.Web.Common/Extensions/MemberClaimsPrincipalExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using System.Security.Claims;
+using Microsoft.AspNetCore.Identity;
+
+namespace Umbraco.Extensions;
+
+public static class MemberClaimsPrincipalExtensions
+{
+
+    /// <summary>
+    /// Tries to get specifically the member identity from the ClaimsPrincipal
+    /// <remarks>
+    /// The identity returned is the one with default authentication type.
+    /// </remarks>
+    /// </summary>
```

Comment:
```suggestion
    /// <summary>
    /// Tries to get specifically the member identity from the ClaimsPrincipal
    /// </summary>
    /// <remarks>
    /// The identity returned is the one with default authentication type.
    /// </remarks>
```

Just saw this had got nested incorrectly.